CHANGELOG.md

```diff
@@ -23,16 +19,19 @@
 ### Development
 
 ### Raw diffs
-- [capa v7.0.0-beta...master](https://github.com/mandiant/capa/compare/v7.0.0-beta...master)
-- [capa-rules v7.0.0-beta...master](https://github.com/mandiant/capa-rules/compare/v7.0.0-beta...master)
+- [capa v7.0.0...master](https://github.com/mandiant/capa/compare/v7.0.0...master)
+- [capa-rules v7.0.0...master](https://github.com/mandiant/capa-rules/compare/v7.0.0...master)
+
+## v7.0.0
+This is the v7.0.0 release of capa which was mainly worked on during the Google Summer of Code (GSoC) 2023. A huge
+shoutout to our GSoC contributors @colton-gabertan and @yelhamer for their amazing work.
```

Comment:
lets add links to blog posts when they go live.

ditto with the main readme.

Reply:
yes, I will update the Release page with the links